salt/_pillar/metalk8s_private.py

```diff
@@ -14,36 +14,39 @@ def __virtual__():
     return __virtualname__
 
 
+def _read_private_key(key_name, key_path):
+    try:
+        with open(key_path, "r") as key_file:
+            key_data = key_file.read()
+    except IOError as exn:
+        # File not found
```

Comment:
IMHO, `ENOENT` is clear enough (that's the point of importing `errno` to use the named constant instead of `2`)